package.json

```diff
@@ -5119,7 +5123,7 @@
           "razorComponentElement": [
             "entity.name.class.element.component"
           ],
-          "razorComponentAttribute": [
+          "RazorComponentAttribute": [
```

Comment:
same here - this feels really inconsistent.  Is it possible to make these all uniform